webextensions/api/bookmarks.json

```diff
@@ -339,7 +339,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/bookmarks/onCreated",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "4"
```

Comment:
✅ https://github.com/chromium/chromium/commit/5592d7ec9c8b63cca9ab980d4c2d9888dafc3761 in [4.0.212.1](https://github.com/chromium/chromium/releases/tag/4.0.212.1) before 4.0.223.

Reply:
Technically, the feature landed before as `onAdded`, but only after Chrome 3.